www/src/pages/home/HomeApp.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import { Link, Route, Switch } from 'react-router-dom';
-import Jumbotron from './components/Jumbotron';
+import Jumbotron from '../../components/Jumbotron';
```

Comment:
Not happy about the long resolve paths. Internal component library or local presentation components? Not sure what might be best.